src/parse/asp/objects.go

```diff
@@ -944,31 +944,50 @@ func newConfig(state *core.BuildState) *pyConfig {
 		}
 	}
 
-	loadPluginConfig(state.Config, state, c)
+	loadPluginConfig(state, state, c)
```

Comment:
This took me a while to get my head around but I guess it makes sense. There are two distinct cases here right? Either ones a plugin and the other isn't, in which case we're parsing the plugin config in the host-repo context, or we're in a plugin and are parsing the config in the plugin context? Neat!

Reply:
Oh sorry - just read the comment. That's pretty much what you were talking about.